README.md

```diff
@@ -1,34 +1,27 @@
-# ⚡️ SwaySwap
-[![License](https://img.shields.io/github/license/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
-[![Issues Open](https://img.shields.io/github/issues/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
-[![Github Forks](https://img.shields.io/github/forks/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
-[![Github Stars](https://img.shields.io/github/stars/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+# ⚡️ [SwaySwap](https://fuellabs.github.io/swayswap) ⚡️
 
-Short description
-    
-<!-- ![SwaySwap Interface](cover.png) -->
+[![build](https://github.com/FuelLabs/swayswap/actions/workflows/gh-pages.yml/badge.svg)](https://github.com/FuelLabs/swayswap/actions/workflows/gh-pages.yml)
+[![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
+![twitter](https://img.shields.io/twitter/follow/SwayLang?style=social)
 
 ## About the Project
 
-About the project
+SwaySwap is a decentralized application that takes advantage of Fuel's lightning fast SDK, and [Sway](https://github.com/FuelLabs/sway) language. Create a wallet, acquire funds from a faucet and start exchanging assets with the world's fastest modular stack.
```

Comment:
@adlerjohn can I get your review of this paragraph? want to make sure this is an accurate statement.

Reply:
Eh, good enough.